gruntfile.js

```diff
@@ -345,7 +345,6 @@ module.exports = function (grunt) {
                     port: 9000,
                     middleware: function (connect) {
                         return [
-                            delayApiCalls,
```

Comment:
The problem with this is that it makes e2e tests very slow... Maybe we can just trigger the REST call when clicking a button